src/sample_entropy/sample_entropy.jl

```diff
@@ -35,21 +36,22 @@ function computeprobs(x; k::Int, m::Int, r, metric = Chebyshev())
 end
 
 """
-    sample_entropy(x; m = 2, r = StatsBase.std(x), metric = Chebyshev(),
-        normalize = false) → SampEn::Float64
+    SampleEntropy(; r = 0.1, m::Int = 2, normalize = false)
 
 Estimate the sample entropy (Richman & Moorman, 2000)[^Richman2000]
```

Comment:
If this has no `q` and no `base`, I guess we need to rethink our approach of having entropy estimator structures and passing them to `genentropy`.

Reply:
Can you please hault releasing new versions for a moment, because I want to go through the documentation, and there are no docs anyways yet for all this stuff, at least not in DynamicalSystems.jl.

Reply:
> If this has no q and no base, I guess we need to rethink our approach of having entropy estimator structures and passing them to genentropy

Yes, there are a bunch of methods in the literature that estimate "entropy-like" quantities, but mean perhaps something slightly different that explicitly estimating a normalized distribution and then computing (generalized/Tsallis/whatever) entropy. Approximate and sample entropy are among these methods. I remember now that is why I originally didn't include them as estimator structures to begin with.

> Can you please hault releasing new versions for a moment, because I want to go through the documentation, and there are no docs anyways yet for all this stuff, at least not in DynamicalSystems.jl.

Sure, we can keep the current release until we figure stuff out. I merged #76, but didn't release a new version yet. I haven't added any docs to CausalityTools either.

Reply:
> Can you please hault releasing new versions for a moment, because I want to go through the documentation, and there are no docs anyways yet for all this stuff, at least not in DynamicalSystems.jl.

Perhaps we can just restart building the docs here too, but not link them in the Readme.md? I'm building the docs locally for all my PRs, but it would be a bit easier if they were build automatically with the PRs.